CONTRIBUTING.md

```diff
@@ -70,3 +78,14 @@ to include the following:
 
 The maintainers for this repo will review your code and provide feedback. If everything looks good, they will merge the
 code and release a new version, which you'll be able to find in the [releases page](../../releases).
+
+### Bump the minor
```

Comment:
Ah we actually already have this in the README (https://github.com/gruntwork-io/cloud-nuke#choosing-a-new-release-tag), so we should reuse that and link there to avoid duplication.

Reply:
Thanks for finding that! Will make the change.